### PR TITLE
Readme: update example to use duckduckgo.com

### PR DESCRIPTION
**Pull request by rstacruz**
Yahoo.com's home page took a very long time to work—for a moment I thought nightmare.js wasn't working!

nightmare

### CODE REVIEW

**Comment by reinpk** [comment]
good idea, thanks :)

File: Readme.md [hunk 1]
```diff
@@ -30,16 +30,13 @@ Many thanks to [@matthewmueller](https://github.com/matthewmueller) and [@rosshi
 Let's search on Yahoo:
```
Comment (rstacruz) [inline Readme.md]: oops, haha, this needs to change too

Comment (rstacruz) [inline Readme.md]: but i think someone already edited that

**Comment by reinpk** [comment]
Yeah I fixed whole merging ;)

**Pull request merged**